docs/reference/content/whats-new.md

```diff
@@ -7,6 +7,15 @@ title = "What's New"
   pre = "<i class='fa fa-level-up'></i>"
 +++
 
+## What's new in 3.9
+
+Key new features of the 3.9 Java driver release:
+
+## Android support
```

Comment:
I was initially unsure about adding this - as it may prompt more questions about using it standalone. However, I've made a suggestion to the text below.